### DISCUSSION TITLE
Disabling Firmware Retract for SKRME3V3 and issues with compiling

### DISCUSSION THREAD

**Discussion by Aldudur** [mriscoc/Ender3V2S1#404] [Q&A] [answered]
Hi, first of all many thanks for all the work done on this amazing project !

I'm currently using the _Experimental-Ender3V2-SKRME3V3-BLTUBL-LA-20220817_ build and I'd like to disable Firmware Retraction as it's no use for me. (but I'd like to keep the LA, that's why I'm using this version)

But for some reason I run into multiple weird issues when I'm compiling myself the firmware : 
- Quick Menu doesn't show up anymore 
- Unable to save or load any Bed Level Mesh 
- Having weird issues with BL Touch

I'm just adding '//' in front of FWRETRACT and compiling, that's all.

So first question is : have you any idea why my firmware isn't compiling correctly ?

And second question : could you make the **exact** same version of the  _Experimental-Ender3V2-SKRME3V3-BLTUBL-LA-20220817_ build but without the Firmware Retract ?

**Comment by mriscoc** [comment, marked as answer]
Hi, if you do not configure your slicer to use firmware retraction it will not be in use and your printer will use the normal slicer-based retraction.
Firmware retraction enables the printer to understand the special G10 and G11 G-codes.

**Comment by Aldudur** [comment]
Ok thanks for the quick answer !
